packages/react-components/src/components/Select/Select.jsx

```diff
@@ -6,6 +6,10 @@ import { makeField } from '../../helpers/fields';
 
 import dispatchAnalyticsEvent from '../../helpers/analytics';
 
+if (!i18next.exists('collapse-all-aria-label')) {
```

Comment:
I chose this key since it's somewhat unique to the component-library and I don't expect to find it in vets-website.